index-scheduler/src/lib.rs

```diff
@@ -867,7 +867,8 @@ impl IndexScheduler {
     /// Create a new index without any associated task.
     pub fn create_raw_index(&self, name: &str) -> Result<Index> {
         let wtxn = self.env.write_txn()?;
-        let index = self.index_mapper.create_index(wtxn, name)?;
+        let date = Some((time::OffsetDateTime::now_utc(), time::OffsetDateTime::now_utc()));
+        let index = self.index_mapper.create_index(wtxn, name, date)?;
```

Comment:
```suggestion
        let index = self.index_mapper.create_index(wtxn, name, date)?;
```

The caller is supposed to forward you the date from the dump here